<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>161</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="ui_render_layers_grpbx">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>370</width>
     <height>146</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Choose the render images to submit for rendering</string>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="title">
    <string>Layer to render</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QListWidget" name="ui_render_images_lswgt"/>
    </item>
    <item>
     <widget class="QCheckBox" name="ui_upload_only">
      <property name="toolTip">
       <string>Only upload dependencies. Don't submit render job</string>
      </property>
      <property name="text">
       <string>Upload Only</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTreeWidget" name="ui_render_images_trwgt">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>340</width>
     <height>78</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
